Back/Reciplastk.Gateway/Controllers/RolController.cs

```diff
@@ -0,0 +1,133 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Reciplastk.Gateway.DataAccess;
+using Reciplastk.Gateway.Models;
+using Reciplastk.Gateway.Services;
+
+namespace Reciplastk.Gateway.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RolController : ControllerBase
+    {
+        private readonly IRolService rolService;
+        private readonly IMapper mapper;
+
+        public RolController(IRolService _rolService, IMapper _mapper)
+        {
+            rolService = _rolService;
+            mapper = _mapper;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAllRol()
```

Comment:
Controller endpoints are not following the existing pattern. 
This should be returning standard responses 
```suggestion
        public async Task<HttpResponseModel > GetAll()
```